packages/muelu/src/CMakeLists.txt

```diff
@@ -471,6 +471,9 @@ TRIBITS_ADD_LIBRARY(
   SOURCES ${SOURCES_LGN}
   )
 
+target_link_libraries(muelu muelu_lgn)
```

Comment:
FYI: This will break the debian builds of Trilinos which prefix all of the Trilinos library names.  This will also break any future attempt to get installation testing and backward compatibility testing working again where one configures the test sand examples in Trilinos against a pre-installed version of Trilinos and therefore in that case TRIBITS_ADD_LIBRARY() is a no-op and these `target_link_libraries()` command will crash.

In a follow-up PR, please switch to using the [DEPLIBS argument in TRIBITS_ADD_LIBRARY()](https://tribits.org/doc/TribitsDevelopersGuide.html#formal-arguments-tribits-add-library) as that will work in all cases.